### PR TITLE
Airflow Pipeline

### PR DESCRIPTION
**Pull request by r4vi**
Adds `airflow.yaml` pipeline which triggers for
PRs into the [example dags
repo](https://github.com/ministryofjustice/analytics-platform-airflow-example-dags)
in `dev` and the [airflow dags
repo](https://github.com/moj-analytical-services/airflow-dags/) in `alpha`.

### CODE REVIEW

File: airflow.yaml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            run:
+              path: python
+              args:
+                - -c
+                - |
+                  import itertools
+                  from airflow.models import DagBag
+                  import airflow.utils.dag_processing
+                  
+                  d = DagBag()
```
Comment (andyhd) [inline airflow.yaml:71]: Since both linting and listing roles start the same way, is there any need for two separate tasks?

Comment (r4vi) [reply airflow.yaml:71]: it's so they show up as two different checks in the github UI

Comment (andyhd) [reply airflow.yaml:71]: Sure, but why? It could just be one "test" job that raises different error messages

**Review by andyhd — APPROVED** [review]

File: airflow.yaml (hunk 1 above)
Comment (r4vi) [reply airflow.yaml:71]: to save them a click through to the concourse UI

**Pull request merged**